python/CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
 # Change Log
 All notable changes to this project will be documented in this file.
- 
+
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v0.3.0] - September 25, 2024
+
+Summary of changes:
+- [add missing double channel computation](https://github.com/sift-stack/sift/pull/92)
+- [update protos](https://github.com/sift-stack/sift/pull/94)
+- [update protos](https://github.com/sift-stack/sift/pull/95)
+- [csv example ingestion](https://github.com/sift-stack/sift/pull/96)
+- [update example](https://github.com/sift-stack/sift/pull/97)
+- [Add multiple CSV ingestion example](https://github.com/sift-stack/sift/pull/98)
+- [Add metadata arg to use_sift_channel](https://github.com/sift-stack/sift/pull/99)
```

Comment:
Think it's worth going into any of these in any more detail or no?

Reply:
we good!